lang/python/homeassistant/files/hass.init

```diff
@@ -0,0 +1,17 @@
+#!/bin/sh /etc/rc.common
+
+USE_PROCD=1
+START=99
+STOP=10
+
+PROG=/usr/bin/hass
+CONF=/srv/homeassistant
+
+start_service() {
+        procd_open_instance
+        procd_set_param command "$PROG" -c "$CONF"
```

Comment:
if you run this with `procd_set_param command "$PROG" -c "$CONF" --daemon`  would it work to remove 
```
procd_set_param stdout 1
procd_set_param stderr 1`
```
?

Reply:
Why do you want to remove standard output redirection to syslog? I think that it is a good approach. I don't see reason why we should avoid it.

Also parameters you posted are not good idea as well. Using `--daemon` with procd service is bad. Procd looses track of service if it does double fork (which is what `--daemon` instructs it to do). This means that there is effectively no way to kill service trough procd. It is something I encountered with syslog-ng recently so I am clearly aware that it is not a good idea.

Reply:
oh right;
i forgot that procd requires services in foreground [been a while since i used it more thoroughly]
the `--daemon` arg is stupid to use now; so, my bad

regarding:
> Why do you want to remove standard output redirection to syslog? I think that it is a good approach. I don't see reason why we should avoid it.

things may have changed in a while;
but, personally, i liked it when services would use syslog directly, so that procd would not need to do any output redirection to syslog;

i sometimes found that procd's output redirection was sometimes buggy [but that probably isn't the case anymore];

some services, do allow the program to stay in foreground, while output-ing directly to syslog
[ i was looking for this when i suggested `--daemon` ]

Reply:
I'd use procd_append_param. Although that's just a style thing.